.github/workflows/e2e-latest.yml

```diff
@@ -54,6 +54,14 @@ jobs:
         IMAGE_PATH=gcr.io/${{ env.PROJECT_ID }}/tests/gcr.io/stackdriver-sandbox-230822/sandbox/loadgenerator/gke:$GITHUB_SHA
         docker build -t $IMAGE_PATH ./src/loadgenerator
         docker push $IMAGE_PATH
+    - name: Validate SRE Recipes Configs
```

Comment:
This might do better in the standard [CI workflow](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/main/.github/workflows/ci.yml). The E2E workflow is used by tests that need to run against a live sandbox environment, but I don't think these tests should need that

Reply:
Moved it out to ci.yaml.

It took quite some try because there are installation issues for pip with python3.5 that's running in ci.yaml. We should probably upgrade the default python version in the self hosted containers that's running the CI workflow.